proxy/proxy.go

```diff
@@ -1196,10 +1197,12 @@ func (p *Proxy) errorResponse(ctx *context, err error) {
 	}
 
 	id := unknownRouteID
+	backendType := unknownRouteBackendType
 	backend := unknownRouteBackend
 	if ctx.route != nil {
 		id = ctx.route.Id
-		backend = ctx.route.Backend
+		backendType = ctx.route.BackendType.String()
+		backend = ctx.request.URL.Host
```

Comment:
can we please include the scheme from ctx.request.URL?

Reply:
Done